src/config.ts

```diff
@@ -6,43 +6,54 @@ const Page: z.ZodType<Page> = z.any();
 
 export const configSchema = z.object({
   /**
+   * **Required:**
```

Comment:
Same from my comment on README 😉

Reply:
I used the same convention as mentioned for the README.md file for the src/config.ts file as well. I will add more details for parameters, examples, defaults, etc. as we progress.